From 20296a0f48c670f052ca294ae6060b57fde6a76e Mon Sep 17 00:00:00 2001
From: Song Zhao <song.zhao@amlogic.com>
Date: Mon, 4 May 2020 16:58:06 -0700
Subject: [PATCH] update to patch 37 of CL99272

On the base of
0004-v4l2-support-compressed-buffer-for-h265-vp9-dec-1-1.patch
---
 drivers/frame_provider/decoder/mpeg12/vmpeg12_multi.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/frame_provider/decoder/mpeg12/vmpeg12_multi.c b/drivers/frame_provider/decoder/mpeg12/vmpeg12_multi.c
index 207f2633..b28cfa0b 100644
--- a/drivers/frame_provider/decoder/mpeg12/vmpeg12_multi.c
+++ b/drivers/frame_provider/decoder/mpeg12/vmpeg12_multi.c
@@ -489,9 +489,6 @@ static int vmpeg12_v4l_alloc_buff_config_canvas(struct vdec_mpeg12_hw_s *hw, int
 	hw->canvas_config[i][1].endian		=
 		(hw->canvas_mode == CANVAS_BLKMODE_LINEAR) ? 7 : 0;
 	canvas_config_config(canvas_u(canvas), &hw->canvas_config[i][1]);
-	/* mpeg2 decoder canvas need to be revert to match display canvas */
-	hw->canvas_config[i][1].endian		=
-		(hw->canvas_mode != CANVAS_BLKMODE_LINEAR) ? 7 : 0;
 
 	debug_print(DECODE_ID(hw), PRINT_FLAG_BUFFER_DETAIL,
 		"[%d] %s(), canvas: 0x%x mode: %d y: %x uv: %x w: %d h: %d\n",
-- 
2.24.1

